test/NuGet.Core.Tests/NuGet.Commands.Test/RuntimePackageTests.cs

```diff
@@ -116,7 +116,7 @@ public async Task RuntimePackage_RejectedPackagesAreNotMerged()
                 packages.Add(packageX1Runtime);
                 packages.Add(packageX2Runtime);
 
-                SimpleTestPackageUtility.CreatePackagesAsync(packages, repository);
+                await SimpleTestPackageUtility.CreatePackagesAsync(packages, repository);
```

Comment:
Good catch.  Did this result in test flakiness?

Reply:
Yeah. 

So I created this issue to enable warnings as errors in tests. 

https://github.com/NuGet/Home/issues/8009